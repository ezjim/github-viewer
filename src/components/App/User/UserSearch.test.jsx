import React from 'react';
import { shallow } from 'enzyme';
import UserInput from './UserSearch';

describe('UserInput component', () => {
  it('renders UserInput', () => {
    const wrapper = shallow(<UserInput 
      username="ezjim"
      onUserChange={() => {}}
      onUserSubmit={() => {}}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
