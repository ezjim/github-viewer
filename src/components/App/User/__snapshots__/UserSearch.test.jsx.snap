// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInput component renders UserInput 1`] = `
<Fragment>
  <input
    name="username"
    onChange={[Function]}
    type="text"
    value="ezjim"
  />
  <input
    name="search"
    onClick={[Function]}
    type="button"
    value="Search"
  />
</Fragment>
`;
